app/src/main/java/com/android/partagix/ui/components/ItemUi.kt

```diff
@@ -132,7 +132,7 @@ fun ItemUi(
           .animateContentSize(
               animationSpec = tween(durationMillis = 300, easing = LinearOutSlowInEasing))
           .background(
-              color = MaterialTheme.colorScheme.onPrimary, shape = RoundedCornerShape(size = 4.dp))
+              color = MaterialTheme.colorScheme.onPrimary, shape = RoundedCornerShape(size = 8.dp))
```

Comment:
This change is false, the background color was rounded correctly before, and now there is a tiny space (see picture below).

![image](https://github.com/EPFLSWENT2024G1/partagix/assets/126057141/52832744-e01d-411a-b612-5bce9b297c32)